include/rocksdb/utilities/backupable_db.h

```diff
@@ -24,6 +24,11 @@
 
 namespace ROCKSDB_NAMESPACE {
 
+// The default DB file checksum function name.
+constexpr char kDbFileChecksumFuncName[] = "FileChecksumCrc32c";
+// The default BackupEngine file checksum function name.
+constexpr char kBackupFileChecksumFuncName[] = "crc32c";
```

Comment:
Also here

Reply:
These will almost certainly be refactored soon, so NBD to me. :)